.gitignore

```diff
@@ -0,0 +1,204 @@
+# version file generated by setuptools-scm
+/vllm/_version.py
+
+# vllm-flash-attn built from source
+vllm/vllm_flash_attn/
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
```

Comment:
Do we really need this huge .gitignore directly lifted from vLLM?

Reply:
Looks like a generic .gitignore for python for the most part...
I just removed vllm-related stuff, should I try to remove all that are not strictly necessary?

Reply:
I guess it is fine.